deploy_monitoring.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: deploy control plane monitoring
+  hosts: control-plane
+  connection: local
+  gather_facts: false
+  vars:
+    setup_yaml:
+      - url: https://raw.githubusercontent.com/metal-stack/releases/{{ metal_stack_release_version }}/release.yaml
+        meta_var: metal_stack_release
+  roles:
+    - name: ansible-common
+      tags: always
+    - name: monitoring
```

Comment:
This role needs to come from the `metal-roles` repository. You can open a PR there for this role and test it by adding the following line to `group_vars/all/images.yaml`:

```
metal_roles_version: <name-of-your-branch-in-metal-roles-repo>
```